banyand/metadata/schema/index.go

```diff
@@ -19,16 +19,19 @@ package schema
 
 import (
 	"context"
+	"hash/crc32"
 
 	"google.golang.org/protobuf/proto"
 
 	commonv1 "github.com/apache/skywalking-banyandb/api/proto/banyandb/common/v1"
 	databasev1 "github.com/apache/skywalking-banyandb/api/proto/banyandb/database/v1"
+	"github.com/pkg/errors"
```

Comment:
the location of this import line is wrong

Reply:
forget to run `make pre-push`